integration_test/third_party_apps_test.go

```diff
@@ -222,6 +253,102 @@ func runLoggingTestCases(ctx context.Context, logger *logging.DirectoryLogger, v
 	return err
 }
 
+func runMetricsTestCases(ctx context.Context, logger *logging.DirectoryLogger, vm *gce.VM, testCaseBytes []byte) error {
+	var entries []expectedMetricEntry
+	var requiredEntries []expectedMetricEntry
+	err := yaml.UnmarshalStrict(testCaseBytes, &entries)
+	if err != nil {
+		return fmt.Errorf("could not unmarshal contents of expected_metrics.yaml: %v", err)
+	}
+	if len(entries) == 0 {
+		logger.ToMainLog().Printf("Skipping metrics test: expected_metrics.yaml is empty")
+		return nil
+	}
```

Comment:
I think we can safely skip the check for the empty file, especially since they can just use a missing file to express the same thing. I guess there's little harm in this code for handling an empty file, but it feels strange to have both ways to express "skip this test please".

Reply:
I think the issue here is that without this check, you'd get "found expected_metrics.yaml, running metrics test cases...", and then nothing…

Reply:
I was tempted to actually make this an error for that reason, since the intended mechanism for skipping is to not have the file at all, and having an empty file is possibly an oversight by the author who might've forgotten to populate it. WDYT?

Reply:
igor, IIUC you would have gotten an error that there is no representative metric in the file. I think that's all right, an error specifically for an empty file would be all right too.

Reply:
Changed to error on empty file.